netbeans.apache.org/src/content/about/history.asciidoc

```diff
@@ -0,0 +1,166 @@
+////
+     Licensed to the Apache Software Foundation (ASF) under one
+     or more contributor license agreements.  See the NOTICE file
+     distributed with this work for additional information
+     regarding copyright ownership.  The ASF licenses this file
+     to you under the Apache License, Version 2.0 (the
+     "License"); you may not use this file except in compliance
+     with the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing,
+     software distributed under the License is distributed on an
+     "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+     KIND, either express or implied.  See the License for the
+     specific language governing permissions and limitations
+     under the License.
+////
+= Apache NetBeans History
+:jbake-type: page
+:jbake-tags: about
+:jbake-status: published
+:keywords: Apache NetBeans History
+:description: Apache NetBeans History
+:toc: left
+:toc-title:
+:icons: font
+
+== Apache NetBeans History
+
+NetBeans started as a student project (originally called Xelfi) in
+Czechoslovakia, now the Czech Republic, in 1996. The goal was to write a
+Delphi-like Java IDE (Integrated Development Environment) in Java. Xelfi was
+the first Java IDE written in Java, with its first pre-releases in 1997. Xelfi
+was a fun project to work on, especially since the Java IDE space was uncharted
+territory at that time.
+
+image::nb-history-1.png[caption="", title="Here you see the original group of students who constituted the original NetBeans Team"]
+
+The project attracted enough interest that the students, once they graduated,
+decided that they could market it as a commercial product. Soliciting resources
+from friends and relatives for a website, they formed a company around it.
```

Comment:
We were selling Xelfi as shareware. There was no real company that would deserve such title.

We haven't sold much copies, but we attracted attention of Roman Staněk...